cypress/support/test-utils.ts

```diff
@@ -95,3 +102,21 @@ function verifyHeaderContent(content: {
       }
     })
 }
+
+// Function to verify the content's title and description
+function verifyContent(content: {
+  title: string
+  description: string | string[]
+}) {
+  cy.get('h1').should('have.text', content.title)
+
+  if (content.description) {
+    if (Array.isArray(content.description)) {
+      content.description.forEach((text: string) => {
+        cy.get('p').should('contain.text', text)
+      })
+    } else {
+      cy.get('p').should('contain.text', content.description)
+    }
+  }
+}
```

Comment:
Why does we have `description` here? If this is meant to check pages without a PageHeader - like Privacy and Terms - then, we only check for the title, right? Or did I miss something?